articles/container-instances/container-instances-tutorial-deploy-app.md

```diff
@@ -47,7 +47,14 @@ Container registry password:
 az acr credential show --name <acrName> --query "passwords[0].value"
 ```
 
-To deploy your container image from the container registry with a resource request of 1 CPU core and 1 GB of memory, run the following command. Replace `<acrLoginServer>` and `<acrPassword>` with the values you obtained from the previous two commands. Replace `<acrName>` with the name of your container registry.
+Your application will need to have
+been [prepared in advance][prepare-app]; to deploy your container image from the container registry with a
+resource request of 1 CPU core and 1 GB of memory, run the following
+command [`az container create`][az-container-create]. Replace
```

Comment:
`run the following [az container create][az-container-create] command.`

(Move "command" to last word of sentence, and remove backticks around command to keep in line with convention in the rest of the docs.)

Reply:
:+1: